README.md

```diff
@@ -15,75 +15,74 @@
 
-| Package                                                                         | NPM                                                                                                       |
-| ------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
-| [Hops Server](https://github.com/xing/hops/tree/master/packages/server)         | [![npm](https://img.shields.io/npm/v/hops-server.svg)](https://www.npmjs.com/package/hops-server)         |
-| [Hops Middleware](https://github.com/xing/hops/tree/master/packages/middleware) | [![npm](https://img.shields.io/npm/v/hops-middleware.svg)](https://www.npmjs.com/package/hops-middleware) |
-| [Hops Renderer](https://github.com/xing/hops/tree/master/packages/renderer)     | [![npm](https://img.shields.io/npm/v/hops-renderer.svg)](https://www.npmjs.com/package/hops-renderer)     |
+There are two main reasons. Firstly, snippets get outdated. Hops, however, is committed to best practices and evolve with them. Secondly, Hops is organized in modules that need to run both on the client and on the server. Keeping a track of both sides and at the same time maintaining an in sync version of a client side and server side snippet per module can be really tedious. Using a framework like Hops takes away that pain.
+
+#### Why does Hops not support "eject"?
+
+[Create React App](https://github.com/facebook/create-react-app) offers you the option to "eject". This means dumping Webpack configuration files and scripts, leaving you with a stand-alone project. We Hops people do not think, this is a good idea. Those files still force their philosophy on you while at the same time they are hard to understand. We thus do not believe this to be a practical way to overcome vendor lock-in. Hops instead allows you to customize either by configuration or adding modules.
+
+#### Can we trust in Hops to be around for the forseeable future?
+
+Just as you might put trust into React as Facebook has committed itself to it and eats its own dog food, Hops is being backed and used by a European professional networking site.
+
+#### Why so many modules?
+
+First of all some modules are separated from others for ease of development, but can be seen as one logical entity by you. Like `react`, `react-dom`, and `prop-types`. It also enables you to just have a minimal set of modules for each setting. E.g. when deploying your app, you do not need the CLI or the build module. Hops core modules are thus split between build and deploy.
+
+There are more modules in the Hops repository except for the core, but they are merely options to give you choices.
+
+#### I like Hops, but I want to use something that is not quite mainstream
+
+Hops allows you to write a package of your own and plug it into Hops' build process. This has been proven to actually work. E.g. people have done this for TypeScript.
```

Comment:
This sentence reads like there should be a list of things, but it ends with just one thing - how about adding "styled-components" or just an "etc"?